### DISCUSSION TITLE
How to start using pyopenxr

### DISCUSSION THREAD

**Discussion by vstadnytskyi** [cmbruns/pyopenxr#83] [General]
Hi,

I followed instruction on the readme page and I am getting an error. Does it mean I need to install something else?

![image](https://user-images.githubusercontent.com/29989124/165384724-345b01e9-9c33-4f86-be25-bb5ff98ceea5.png)

**Comment by cmbruns** [comment]
It looks like your system does not support OpenGL. What sort of system are you running on? What are the contents of the list of available extensions?
